src/js/providers/video-listener-mixin.js

```diff
@@ -50,6 +50,10 @@ const VideoListenerMixin = {
         }
         this._helperLastVideoHeight = height;
 
+        if (this.seeking) {
+            return;
+        }
```

Comment:
This conflicts with the statement below `if (this.state === STATE_PLAYING || this.seeking) {`. 

I think what you want is to add !seeking as a guard around here, but we might not need it anymore.

`if (!this.video.paused && (this.state === STATE_STALLED || this.state === STATE_LOADING)) {`